models/tokenprice/tokenprice_dfkgold.sql

```diff
@@ -0,0 +1,76 @@
+{{ 
+    config(
+        materialized='incremental',
+        unique_key = 'timestamp',
+        incremental_strategy = 'delete+insert',
+        tags=['tokenprice'],
+        cluster_by=['timestamp']
+    )
+}}
+
+with
+
+historical_prices as (
+    select
+        date_trunc('day',timestamp) as timestamp,
+        avg(price) as price
+    from harmony.dev.ant_token_prices
+    where {{ incremental_load_filter_withdays("timestamp", -5) }}
+    and token_address = '0x3a4edcf3312f44ef027acfd8c21382a5259936e7' -- gold
+    group by 1
+    order by 1 desc
```

Comment:
maybe no need to order here as you do ordering in the final query using the cluster by logic above